apps/dashboard/src/components/engine/configuration/engine-wallet-config.tsx

```diff
@@ -1,102 +1,80 @@
+            href={`/dashboard/engine/${instance.id}`}
+            className="text-link-foreground hover:text-foreground"
           >
-            Learn more about backend wallets
-          </Link>
-          .
-        </Text>
+            Overview
+          </Link>{" "}
+          tab. To use other wallet types, configure them below.
+        </p>
       </Flex>
-      <Flex flexDir="column" gap={4}>
-        <ButtonGroup size="sm" variant="outline" spacing={2}>
-          <Button
-            type="button"
-            leftIcon={
-              <Icon
-                as={
-                  selected === "local"
-                    ? MdRadioButtonChecked
-                    : MdRadioButtonUnchecked
-                }
-                boxSize={6}
-              />
-            }
-            borderColor={selected === "local" ? "unset" : undefined}
-            colorScheme={selected === "local" ? "blue" : undefined}
-            py={6}
-            rounded="lg"
-            onClick={() => setSelected("local")}
-          >
-            Local Wallet
-          </Button>
-          <Button
-            type="button"
-            leftIcon={
-              selected === "aws-kms" ? (
-                <MdRadioButtonChecked />
-              ) : (
-                <MdRadioButtonUnchecked />
-              )
-            }
-            borderColor={selected === "aws-kms" ? "unset" : undefined}
-            colorScheme={selected === "aws-kms" ? "blue" : undefined}
-            py={6}
-            rounded="lg"
-            onClick={() => setSelected("aws-kms")}
-          >
-            AWS KMS
-          </Button>
-          <Button
-            type="button"
-            leftIcon={
-              selected === "gcp-kms" ? (
-                <MdRadioButtonChecked />
-              ) : (
-                <MdRadioButtonUnchecked />
-              )
-            }
-            borderColor={selected === "gcp-kms" ? "unset" : undefined}
-            colorScheme={selected === "gcp-kms" ? "blue" : undefined}
-            py={6}
-            rounded="lg"
-            onClick={() => setSelected("gcp-kms")}
-          >
-            Google KMS
-          </Button>
-        </ButtonGroup>
 
-        {selected === "local" && <LocalConfig instanceUrl={instanceUrl} />}
-        {selected === "aws-kms" && <KmsAwsConfig instanceUrl={instanceUrl} />}
-        {selected === "gcp-kms" && <KmsGcpConfig instanceUrl={instanceUrl} />}
-      </Flex>
+      <TabButtons
+        tabs={EngineBackendWalletOptions.map(({ key, name }) => ({
+          key,
+          name,
+          isActive: activeTab === key,
+          isEnabled: true,
+          onClick: () => setActiveTab(key),
+          icon:
+            (key === "aws-kms" && !isAwsKmsConfigured) ||
+            (key === "gcp-kms" && !isGcpKmsConfigured)
+              ? ({ className }) => (
+                  <ToolTipLabel label="Not configured">
+                    <CircleAlertIcon className={className} />
```

Comment:
This icon is too large for alert icon, set `size-4` instead
 
```ts
<CircleAlertIcon className={cn(className, 'size-4')} />
```

Reply:
I don't think the tab content needs a warning. Once they're in the content they should know they're adding/updating config.